src/components/PersonCard/PersonCard.tsx

```diff
@@ -0,0 +1,138 @@
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+import { Picture } from '../Picture';
+import { Link } from '../Routing';
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  personCard: {
+    maxWidth: '247px',
+    paddingTop: spacing(1),
+  },
+  cardContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  personName: {
+    paddingBottom: spacing(0.5),
+  },
+  personImage: {
+    width: '86px',
+    height: '86px',
+    paddingBottom: spacing(2),
+  },
+  roleName: {
+    marginBottom: spacing(1),
+  },
+  organizationName: {
+    marginBottom: spacing(0.25),
+  },
+  statusCircle: {
+    height: '8px',
+    width: '8px',
+    marginRight: spacing(1),
+    borderRadius: '50%',
+    display: 'inline-block',
+  },
+  cardActions: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: spacing(2),
+    border: '0.5px solid #e0e0e0',
+  },
+  activeColor: {
+    backgroundColor: palette.primary.main,
+  },
+  notActiveColor: {
+    // TODO not sure exactly what color this should be
+    backgroundColor: palette.error.main,
+  },
+}));
+
+export interface PersonCardProps {
+  name: string;
+  organization: string;
+  role: string;
+  active: boolean;
+  personRoute: string;
+  imageSource: string;
+}
+
+export const PersonCard: FC<PersonCardProps> = ({
+  name,
+  organization,
+  role,
+  active,
+  personRoute,
+  imageSource,
+}) => {
+  const {
+    personCard,
+    personName,
+    statusCircle,
+    cardContent,
+    personImage,
+    roleName,
+    organizationName,
+    cardActions,
+    activeColor,
+    notActiveColor,
+  } = useStyles();
+
+  console.log('ACTIVE: ', active, activeColor);
+
+  return (
+    <Card className={personCard}>
+      <CardContent className={cardContent}>
+        <Picture className={personImage} source={imageSource} />
+        <Typography
+          color="textPrimary"
+          variant="subtitle2"
+          className={personName}
+        >
+          {name}
+        </Typography>
+        <Typography
+          color="primary"
+          variant="caption"
+          className={organizationName}
+        >
+          {organization}
+        </Typography>
+        <Typography
+          color="textSecondary"
+          variant="caption"
+          className={roleName}
+        >
+          {role}
+        </Typography>
+        <Typography color="textSecondary" variant="caption">
+          <Box
+            component="span"
+            className={clsx(statusCircle, {
+              [activeColor]: active,
+              [notActiveColor]: !active,
+            })}
+          />
+          {/* TODO does this text chage when inactive */}
+          Active
+        </Typography>
+      </CardContent>
+      <CardActions className={cardActions}>
+        <Link external={false} to={personRoute}>
```

Comment:
`external` defaults to false, no need to include it. Also let's use a button here instead.
```suggestion
        <ButtonLink to={personRoute}>
```